sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/OpinionSentiment.java

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.models;
+
+import com.azure.core.annotation.Immutable;
+
+/**
+ * The {@link OpinionSentiment} model.
+ */
+@Immutable
+public final class OpinionSentiment {
+    private final String text;
+    private final TextSentiment sentiment;
+    private final SentimentConfidenceScores confidenceScores;
+    private final boolean isNegated;
+    private final int length;
+    private final int offset;
+
+    /**
+     * Create an {@link OpinionSentiment} model that describes opinion sentiment.
+     *
+     * @param text The opinion text property.
+     * @param sentiment The text sentiment label: POSITIVE, NEGATIVE.
+     * @param offset The offset of opinion text.
+     * @param length The length of opinion text.
+     * @param isNegated The boolean indicator to show if the opinion text negated or not.
+     * @param confidenceScores The {@link SentimentConfidenceScores}.
```

Comment:
does the param name need to be plural, it kind of suggests its returning more than one value?

Reply:
Yup. It need to be plural, It keeps consistency as previous did in SentimentConfidenceScores and also consistency cross langauges.
```
private final SentimentConfidenceScores confidenceScores;
```

In .NET api design: https://apiview.dev/Assemblies/Review/b196a5e53fa2441f92e3b1accefa35e7?diffRevisionId=b3605f7e99b744ba8175087b0c933fe2#Azure.AI.TextAnalytics.OpinionSentiment

Reply:
it does return more than one value as it is scores per sentiment label